templates/.github/pull_request_template.md.tpl

```diff
@@ -24,6 +24,10 @@
 <!-- Notes that may be helpful for anyone reviewing this PR -->
 ## Notes for your reviewers
 
+{{- $extraNotes := (stencil.GetModuleHook "reviewerExtraNotes") }}
```

Comment:
Instead of adding a module hook for reviewer extra notes, I think it makes more sense to have an entry in the `CONTRIBUTING.md` for things like "gotchas" in PRs. Otherwise, it seems like this tends to become a large boilerplate that makes PRs more messy

For context, we've been talking about reducing/eliminating most of this PR template.

Reply:
For the first part ideally the `CONTRIBUTING.md` file would be linked in the PR template in this case.

Reply:
I am afraid that the file will be ignored most times while reviewing the PR.

Reply:
cc: @george-e-shaw-iv @zaquestion

Reply:
It 100% will be ignored. This is already a pretty accepted pattern across multiple repos at the company

Reply:
The only repos I know of, that do it... are outreach and outreach-terraform... which are rarely checked off/actually used. Even [outreach](https://github.com/getoutreach/outreach/blob/master/.github/PULL_REQUEST_TEMPLATE.md#pr-guidelines) itself links out to a doc and [client does as well](https://github.com/getoutreach/client/blob/master/.github/pull_request_template.md).

Aside from that, a PR template should never be considered a forcing function. That's what tests are for.

The open-source convention is to use CONTRIBUTING.md for this purpose. We could detach this from stencil-base and make an outreach specific pr template if that's the "company standard" instead, but there's probably a need for more conversation around that than on this PR first.

Reply:
Okay w/e just put it into the contributing doc then @bhogatemadhura if this has production implications in the future at least we'll have something to point towards and ask people why they didn't read it

Reply:
LOL ... this is as good as not doing the work ...

I bet its going to get ignored ... sad reality

Reply:
fixed

Reply:
I'm not a fan of PR templates in general. I've never seen them widely adhered to, and they tend to be too complicated to be very useful.

That said, I think that it _can_ be a useful way to surface info - and that's what the latest change does (call out a supporting doc).